src/eeprom.c

```diff
@@ -27,26 +27,30 @@
 
 #define EEADDR (0x50<<1)
 
-void EEPROM_Init( void ) {
+void EEPROM_Init(void) {
 #define DUMP_EEPROM
 #ifdef DUMP_EEPROM
 	uint8_t dumpbuf[256];
 	EEPROM_Read(dumpbuf, 0, sizeof(dumpbuf));
 	printf("\nEEPROM contents:");
-	for(int foo=0; foo < sizeof(dumpbuf); foo++) {
-		if( (foo & 0x0f) == 0 ) printf("\n0x%04x:",foo);
-		printf(" %02x",dumpbuf[foo]);
+	for (int i = 0; i < sizeof(dumpbuf); i++) {
+		if ((i & 0x0f) == 0){
+			printf("\n0x%04x:", i);
+		}
+		printf(" %02x", dumpbuf[i]);
 	}
```

Comment:
might be more concise to move the dump code to a function and call that function in `#ifdef DUMP_EEPROM` conditional.